### PR TITLE
Lots of small fixes to tests and ci stuff

### PR DESCRIPTION
**Pull request by mmlb**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ clone:
 
 matrix:
   PYTHON_VERSION:
-    - 3.5
```
Comment (mikemrm) [inline .drone.yml:7]: I believe the version of python osie is running still uses 3.5 :/

File: .drone.yml [hunk 2]
```diff
@@ -11,11 +11,10 @@ matrix:
 
 pipeline:
   fmt_and_lint:
-    image: python:3.8-alpine
+    image: nixos/nix:2.3.6
```
Comment (mikemrm) [inline .drone.yml:14]: i feel this over complicates the build process as well as gives the assumption that you need nix to run these tests.

**Review by mikemrm — APPROVED** [review]

File: .drone.yml (hunk 1 above)
Comment (mmlb) [reply .drone.yml:7]: wat

Comment (mmlb) [reply .drone.yml:7]: wow we need to update those containers already

File: .drone.yml (hunk 2 above)
Comment (mmlb) [reply .drone.yml:14]: I can make clear that we don't require nix to run the tests? It does neatly pin the black/pylama(and co) versions which is nice for CI.

File: .drone.yml (hunk 1 above)
Comment (mmlb) [reply .drone.yml:7]: put it back and added a comment

**Review by mikemrm — APPROVED** [review]

**Pull request merged**